commerce-backend-ui-1/ext.config.yaml

```diff
@@ -29,18 +29,12 @@ runtimeManifest:
             LOG_LEVEL: debug
             AIO_CLI_ENV: $AIO_CLI_ENV
             COMMERCE_BASE_URL: $COMMERCE_BASE_URL
-            # Option 1. IMS authentication: https://developer.adobe.com/commerce/extensibility/starter-kit/checkout/connect/#configure-adobe-identity-management-service-ims
             OAUTH_CLIENT_ID: $OAUTH_CLIENT_ID
             OAUTH_CLIENT_SECRETS: $OAUTH_CLIENT_SECRETS
             OAUTH_TECHNICAL_ACCOUNT_ID: $OAUTH_TECHNICAL_ACCOUNT_ID
             OAUTH_TECHNICAL_ACCOUNT_EMAIL: $OAUTH_TECHNICAL_ACCOUNT_EMAIL
             OAUTH_SCOPES: $OAUTH_SCOPES
             OAUTH_IMS_ORG_ID: $OAUTH_IMS_ORG_ID
-            # Option 2. Commerce integration: https://developer.adobe.com/commerce/extensibility/starter-kit/checkout/connect/#create-a-commerce-integration
```

Comment:
@kuflower note I removed this params here because that's for adminsdk and it does not make sense to use Commerce integrations when IMS is a prerequisite.